examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2105,623 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
+	}
+	{
+		// Start Aggregation Example 3
+		pipeline := mongo.Pipeline{
+			{
+				{"$unwind", "$items"},
+			},
+			{
+				{"$match", bson.D{
+					{"items.fruit", "banana"},
+				}},
+			},
+			{
+				{"$group", bson.D{
+					{"_id", bson.D{
+						{"day", bson.D{
+							{"$dayOfWeek", "$date"},
+						}},
+					}},
+					{"count", bson.D{
+						{"$sum", "$items.quantity"},
+					}},
+				}},
+			},
+			{
+				{"$project", bson.D{
+					{"dayOfWeek", "$_id.day"},
+					{"numberSold", "$count"},
+					{"_id", 0},
+				}},
+			},
+			{
+				{"$sort", bson.D{
+					{"numberSold", 1},
+				}},
+			},
+		}
+
+		cursor, err := salesColl.Aggregate(ctx, pipeline)
+
+		// End Aggregation Example 3
+
+		require.NoError(t, err)
+		defer cursor.Close(ctx)
+		requireCursorLength(t, cursor, 4)
+	}
+	{
+		// Start Aggregation Example 4
+		pipeline := mongo.Pipeline{
+			{
+				{"$unwind", "$items"},
+			},
+			{
+				{"$group", bson.D{
+					{"_id", bson.D{
+						{"day", bson.D{
+							{"$dayOfWeek", "$date"},
+						}},
+					}},
+					{"items_sold", bson.D{
+						{"$sum", "$items.quantity"},
+					}},
+					{"revenue", bson.D{
+						{"$sum", bson.D{
+							{"$multiply", bson.A{"$items.quantity", "$items.price"}},
+						}},
+					}},
+				}},
+			},
+			{
+				{"$project", bson.D{
+					{"day", "$_id.day"},
+					{"revenue", 1},
+					{"items_sold", 1},
+					{"discount", bson.D{
+						{"$cond", bson.D{
+							{"if", bson.D{
+								{"$lte", bson.A{"$revenue", 250}},
+							}},
+							{"then", 25},
+							{"else", 0},
+						}},
+					}},
+				}},
+			},
+		}
+
+		cursor, err := salesColl.Aggregate(ctx, pipeline)
+
+		// End Aggregation Example 4
+
+		require.NoError(t, err)
+		defer cursor.Close(ctx)
+		requireCursorLength(t, cursor, 4)
+	}
+	{
+		// Start Aggregation Example 5
+		pipeline := mongo.Pipeline{
+			{
+				{"$lookup", bson.D{
+					{"from", "air_airlines"},
+					{"let", bson.D{
+						{"constituents", "$airlines"}},
+					},
+					{"pipeline", bson.A{bson.D{
+						{"$match", bson.D{
+							{"$expr", bson.D{
+								{"$in", bson.A{"$name", "$$constituents"}},
+							}},
+						}},
+					}}},
+					{"as", "airlines"},
+				}},
+			},
+			{
+				{"$project", bson.D{
+					{"_id", 0},
+					{"name", 1},
+					{"airlines", bson.D{
+						{"$filter", bson.D{
+							{"input", "$airlines"},
+							{"as", "airline"},
+							{"cond", bson.D{
+								{"$eq", bson.A{"$$airline.country", "Canada"}},
+							}},
+						}},
+					}},
+				}},
+			},
+		}
+
+		cursor, err := airAlliancesColl.Aggregate(ctx, pipeline)
+
+		// End Aggregation Example 5
+
+		require.NoError(t, err)
+		defer cursor.Close(ctx)
+		requireCursorLength(t, cursor, 3)
+	}
+}
+
+// RunCommandExamples contains examples of RunCommand operations.
+func RunCommandExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	coll := db.Collection("restaurants")
+
+	err := coll.Drop(ctx)
+	require.NoError(t, err)
+
+	{
+		// Start RunCommand Example 1
```

Comment:
same as aggregation examples

Reply:
See comment above, and lmk what you think 👍